src/lepton/jpgcoder.cc

```diff
@@ -1618,12 +1618,12 @@ class caching_ibytestream
 namespace Sirikata {
 // Implementation of Sirikata::DecoderReader API, using the provided std::vector
 // as input data stream
-class VectorDecoderReader : public DecoderReader {
+class VectorReader : public DecoderReader {
```

Comment:
Let's move this renaming into a separate "fixup" commit and merge it with original where `VectorDecoderReader` and `VectorDecoderWriter` were introduced.